search-app/src/cmr/search/site/data.clj

```diff
@@ -99,7 +112,9 @@
         result (query-exec/execute-query context query)
         {:keys [items granule-counts-map]} result]
     (sort-by :entry-title
-             (map #(assoc % :granule-count (get granule-counts-map (:concept-id %))) items))))
+            (map #(assoc % :granule-count 
+                         (let [count (get-granule-count context (:concept-id %) {:downloadable true})] 
+                           (if (= 0 count) nil count))) items))))
```

Comment:
This can be simplified to `(when-not (zero? gran-count) items)`

Reply:
And you still may want to set the granule count to 0 instead of having a nil in the place of a granule-count on the items.

Reply:
I need a nil because some other function makes nil display "no granules" on the webpage but a count of 0 "browse 0 granules" with a link

Reply:
now changed to ```(when-not (zero? gran-count) gran-count)```